src/main/java/com/fvv/bookstore/util/DateUtil.java

```diff
@@ -0,0 +1,28 @@
+package com.fvv.bookstore.util;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Class DateUtil to convert a String into a Date
```

Comment:
a better comment here could be

Utility class to handle date operations

Reply:
Ok